src/marker.rs

```diff
@@ -665,12 +670,23 @@ impl MarkerExpression {
     /// use std::collections::HashSet;
     /// use std::str::FromStr;
     /// use pep508_rs::MarkerTree;
+    /// use pep440_rs::Version;
     ///
     /// let marker_tree = MarkerTree::from_str(r#"("linux" in sys_platform) and extra == 'day'"#).unwrap();
-    /// assert!(marker_tree.evaluate_extras(&["day".to_string()].into()));
-    /// assert!(!marker_tree.evaluate_extras(&["night".to_string()].into()));
+    /// let versions: Vec<Version> = (8..12).map(|minor| Version::from_release(vec![3, minor])).collect();
+    /// assert!(marker_tree.evaluate_extras_and_python_version(&["day".to_string()].into(), &versions));
+    /// assert!(!marker_tree.evaluate_extras_and_python_version(&["night".to_string()].into(), &versions));
+    ///
+    /// let marker_tree = MarkerTree::from_str(r#"extra == 'day' and python_version < '3.11' and '3.10' <= python_version"#).unwrap();
```

Comment:
Using `unwrap` in examples is discouraged because users may end up copying your code. It's preferred to wrap your test instead

```
rust
# fn main() -> Result<(), Error> {
    ...
# Ok(())
# }
```